modules/tide_content_collection/json/schema.json

```diff
@@ -181,6 +181,14 @@
           "type": "boolean",
           "description": "Store the state of the user interactions in the URL. Only one Content Collection can use this per page."
         },
+        "skipToResultLink": {
+          "type": "boolean",
+          "description": "Show skip to result link when accessing using a tab key, default is true."
+        },
+        "SkipToResultLinkLabel": {
+          "type": "string",
+          "description": "Default is `Skip to results`"
+        },
```

Comment:
Is there some reason that `skipToResultLink` starts with a lowercase letter and `SkipToResultLinkLabel` starts with an uppercase letter? If not, these should be consistent.

Reply:
https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js#L23